### DISCUSSION TITLE
Release Proposal for version 2.8.0

### DISCUSSION THREAD

**Discussion by caipengbo** [apache/kvrocks#2128] [General]
> Refer to https://kvrocks.apache.org/community/create-a-release

It's been almost 3 months since the last release([v2.7.0](https://github.com/apache/kvrocks/releases/tag/v2.7.0)), and many exciting new features and fixes have been merged into the unstable branch. Now I would like to propose the next release version 2.8.0. 

All commits on unstable until https://github.com/apache/kvrocks/commit/db2d244ffd8ef9c5fdb8e4c3c346cef4a195da02 (included) will be considered as commits for this release. The deadline for the cherry-picking period has been set as **March 3, 2024, at 20:00 UTC+8**. No new commits will be accepted after this time.

Feel free to leave your comments about this release proposal. 

cc @apache/kvrocks-committers.

---

Cherry-pick commits:

- #2130 
- #2132 
- #2134

**Comment by Yangsx-1** [comment]
Please don't include commands about stream consumer group, i'll finish them before next version.

**Reply by PragmaTwice** [reply]
Thanks for mention.

We still need to disable XGROUP command in v2.8.0 manually, and you can check the commits of 2.7.0 for how to do (or just cherry-pick that commit). cc @caipengbo

**Comment by git-hulk** [comment]
As @PragmaTwice mentioned in comment: https://github.com/apache/kvrocks/pull/2131#issuecomment-1974792253, we don't pick this PR's commit: https://github.com/apache/kvrocks/pull/2131.

**Comment by caipengbo** [comment]
Thank you all.

I will eventually drop the `XGROUP` command like v2.7 and cherry-pick the following commits:
- #2130 
- #2132 
- #2134

**Reply by git-hulk** [reply]
Thank you!
